.github/workflows/ci.yml

```diff
@@ -128,7 +128,7 @@ jobs:
     runs-on: "ubuntu-20.04"
     strategy:
       matrix:
-        nautobot-version: ["1.0.3", "1.1.6", "1.2.12", "1.3.1"]
+        nautobot-version: ["1.0.3", "1.1.6", "1.2.12", "1.3.6"]
```

Comment:
why not add "latest"?

Reply:
We should also add in a time component to it.